projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -9,6 +9,12 @@
 // We want this interface to be STRICT and cause a compilation error when a removed property is used.
 // Thanks to that, customers using a property that was recently removed, will know they have to adapt their code.
 export interface FeatureTogglesInterface {
+  /**
+   * In `QuoteSummaryActionsComponent` it shows the download button.
+   * API for this button is available in commerce 2211.16 and above
+   */
+  showProposalDownloadButton?: boolean;
```

Comment:
@suprishi, could you, please, rename the toggle to `showDownloadProposalButton`
Thanks!